packages/dds/tree/src/feature-libraries/modular-schema/modularChangeFamily.ts

```diff
@@ -1167,7 +1168,7 @@ interface InvertContext {
 }
 
 interface RebaseTable extends CrossFieldTable<FieldChange> {
-	rebasedFieldToContext: Map<FieldChange, RebaseFieldContext>;
+	fieldToContext: Map<FieldChange, RebaseFieldContext>;
```

Comment:
Would be nice to document that the key is always from the input changeset being rebased.